R/aux_functions.R

```diff
@@ -19,17 +23,45 @@
 #'   incubation_period_samples = c(1, 2, 3, 4, 1),
 #'   alpha = 2
 #' )
-incubation_to_generation_time <- function(incubation_period_samples, alpha) {
+incubation_to_generation_time <- function(incubation_period_samples,
+                                          alpha,
+                                          exposure_time = 0,
+                                          latent_period = 0) {
```

Comment:
would be good to add tests for the new arguments to make sure this works as intended and will do so in the future

Reply:
I've added a couple of unit tests in 33ff394734f05eb95a0eaf99610643372d3fd32a for `incubation_to_generation_time()`. At the moment they only test `latent_period`, I couldn't think of any ways of testing `exposure_time`.